javascript/javascript1/week2/homework-week2/weather-wear.js

```diff
@@ -0,0 +1,15 @@
+function temperatureCheck(temperature) {
```

Comment:
Everything looks ok here. Optionally you could use witch methode for practice you also use an object for clothes which keys are season and values are an array of clothes fx:
`const clothes ={summer:["tShirt","short"], winter: [.....],.....  }`